android/src/main/java/com/geektime/rnonesignalandroid/RNOneSignal.java

```diff
@@ -521,12 +521,12 @@ public void pauseInAppMessages(Boolean pause) {
    public void sendOutcome(final String name, final Callback callback) {
       OneSignal.sendOutcome(name, new OutcomeCallback() {
          @Override
-         public void onSuccess(OutcomeEvent outcomeEvent) {
-            if (outcomeEvent == null)
+         public void onSuccess(OSOutcomeEvent OSOutcomeEvent) {
```

Comment:
the variable should be kept as outcomeEvent, the prefix is only used for the class